Editor/EditorCore/EditorHandleUtility.cs

```diff
@@ -39,9 +40,33 @@ public static bool IsSelectionAppendOrRemoveIfPresentModifier(EventModifiers em)
 
         public static bool IsSelectionPathModifier(EventModifiers em)
         {
-            return (em & EventModifiers.Shift) == EventModifiers.Shift &&
-                ((em & EventModifiers.Control) == EventModifiers.Control ||
-                    (em & EventModifiers.Command) == EventModifiers.Command);
+            var binding = ShortcutManager.instance.GetShortcutBinding("ProBuilder/Selection/Select Path");
+            var combination = binding.keyCombinationSequence.First();
+
+            //At least one modifier is required for the path selection
+            if (combination.modifiers.Equals(ShortcutModifiers.None))
+            {
+                Debug.LogWarning("The shortcut 'ProBuilder/Selection/Select Path' must have at least one modifier, otherwise path selection will not work.");
+                return false;
+            }
```

Comment:
Have you considered going the `ShortcutManager.shortcutBindingChanged` route? This is giving me hundreds of warnings the moment I enter the PB context if I set it to a shortcut without modifier. Could be cleaner if user was notified when they attempt to change the binding.

Reply:
Fixed in [08bc992](https://github.com/Unity-Technologies/com.unity.probuilder/pull/499/commits/08bc992557bd4d7851d5b09e04b45fd30cc40939) .
Indeed a better solution.
I also moved everything related to PathSelection in the same static class at the same time.